### PR TITLE
[RHDEVDOCS-5164]: Add channel to sink docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-5164

Link to docs preview:
https://59074--docspreview.netlify.app/openshift-enterprise/latest/serverless/eventing/channels/connecting-channels-sinks.html

QE review:
- [x] QE has approved this change.

Additional information:
Replaced https://github.com/openshift/openshift-docs/pull/58843

### CODE REVIEW

File: serverless/eventing/channels/connecting-channels-sinks.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="connecting-channels-sinks"]
+= Connecting channels to sinks
+:context: connecting-channels-sinks
+toc::[]
+
+After events have been sent to a channel from an event source or producer, these events can be sent to a sink by using a subscription. Subscriptions are created by configuring a `Subscription` object, which specifies the channel and the sink (also known as a _subscriber_) to deliver events to.
```
Comment (cardil) [inline serverless/eventing/channels/connecting-channels-sinks.adoc:8]:
```suggestion
Before events can be sent to a channel from an event source or producer, these events can be sent to a sink by using a subscription. Subscriptions are created by configuring a `Subscription` object, which specifies the channel and the sink (also known as a _subscriber_) to deliver events to.
```

We shouldn't suggest people should send data to the not-fully configured (linked) event mesh. All connections should be made first, before sending production data. Of course, users might use channels with persistence (like Kafka), but still, we shouldn't suggest sending data to not fully connected system.

**Review by cardil — CHANGES_REQUESTED** [review]

File: serverless/eventing/channels/connecting-channels-sinks.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/eventing/channels/connecting-channels-sinks.adoc:8]: @pierDipi this was part of the reason I suggested moving these topics to group them as more of a dependency on each other - it seemed odd to me to send events to the channel without subscribers even if the idea is for things to be decoupled.

I think this user story and which one is appropriate may need further discussion within the eventing + QE teams to decide which message we want to send to users? Just changing this sentence will make this page not make sense, it won't help to clarify that we should only send events to a channel if there are subscribers.

File: serverless/eventing/channels/connecting-channels-sinks.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="connecting-channels-sinks"]
+= Connecting channels to sinks
+:context: connecting-channels-sinks
+toc::[]
+
+Events that have been sent to a channel from an event source or producer can be forwarded to one or more sinks by using _subscriptions_.
+You can create subscriptions by configuring a `Subscription` object, which specifies the channel and the sink (also known as a _subscriber_) that consumes the events sent to that channel.
```
Comment (gabriel-rh) [inline serverless/eventing/channels/connecting-channels-sinks.adoc:9]:
```suggestion
You can create subscriptions by configuring a `Subscription` object, which specifies the channel, and the sink (also known as a _subscriber_) that consumes the events sent to that channel.
```

Comment (gabriel-rh) [reply serverless/eventing/channels/connecting-channels-sinks.adoc:9]: Just found it hard to parse when reading

**Review by gabriel-rh — APPROVED** [review]
/LGTM

File: serverless/eventing/channels/connecting-channels-sinks.adoc (hunk 2 above)
Comment (abrennan89) [reply serverless/eventing/channels/connecting-channels-sinks.adoc:9]: I think it sort of reads weirdly with the second comma 😅 will leave as is if that's OK

**Comment by maschmid** [comment]
QE approved
